mpv.cpp

```diff
@@ -103,24 +143,29 @@ MpvObject::MpvObject(QQuickItem * parent)
     // Don't stop on audio output issues
     mpv::qt::set_property(mpv, "audio-fallback-to-null", "yes");
 
+    // User-visible application name used by some audio APIs (at least PulseAudio).
+    mpv::qt::set_property(mpv, "audio-client-name", QCoreApplication::applicationName());
+    // User-visible stream title used by some audio APIs (at least PulseAudio and wasapi).
+    mpv::qt::set_property(mpv, "title", QCoreApplication::applicationName());
+
     // Setup the callback that will make QtQuick update and redraw if there
     // is a new video frame. Use a queued connection: this makes sure the
     // doUpdate() function is run on the GUI thread.
-    mpv_gl = (mpv_opengl_cb_context *)mpv_get_sub_api(mpv, MPV_SUB_API_OPENGL_CB);
-    if (!mpv_gl)
-        throw std::runtime_error("OpenGL not compiled in");
-    mpv_opengl_cb_set_update_callback(mpv_gl, MpvObject::on_update, (void *)this);
     connect(this, &MpvObject::onUpdate, this, &MpvObject::doUpdate,
             Qt::QueuedConnection);
 
-    // Setup handling events from MPV
+    // // Setup handling events from MPV
```

Comment:
why?

Reply:
it's a double negative, we need to add one more `//` :))